### PR TITLE
CAL-96 Added Distance Tolerance to Video Admin Plugin

### PR DESCRIPTION
**Pull request by bdeining**
#### What does this PR do?
This PR adds the Distance Tolerance field to the Video Admin plugin
#### Who is reviewing it?
@glenhein @jlcsmith @lcrosenbu @kcwire 
#### How should this be tested?
Build Alliance Experimental, configure an FMV Stream via the Video Admin plugin, and verify that the Distance Tolerance is set and updated correctly
#### What are the relevant tickets?
https://codice.atlassian.net/browse/CAL-96
#### Screenshots (if appropriate)
https://codice.atlassian.net/browse/CAL-96
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

File: catalog/video/video-mpegts-stream/src/main/java/org/codice/alliance/video/stream/mpegts/netty/UdpStreamProcessor.java [hunk 1]
```diff
@@ -329,7 +336,6 @@ public void checkForRollover() {
     }
 
     public void doRollover(File tempFile) {
-        LOGGER.debug("performing video chunk rollover: tempFile={}", tempFile);
```
Comment (glenhein) [inline catalog/video/video-mpegts-stream/src/main/java/org/codice/alliance/video/stream/mpegts/netty/UdpStreamProcessor.java]: Why are you removing this debug log statement?

Comment (bdeining) [inline catalog/video/video-mpegts-stream/src/main/java/org/codice/alliance/video/stream/mpegts/netty/UdpStreamProcessor.java]: Hmm, must have accidentally done that

**Comment by glenhein** [comment]
I set the distance tolerance in the admin page to 0.1. The log file reflected that the monitor was started with the correct distance tolerance. But when I opened the configuration page for the UDP monitor, the distance tolerance was the original value. So it looks like it wasn't updated with the new distance tolerance.

Log entry that shows the correct distance tolerance:

13:23:38,501 | INFO  | ool-137-thread-1 | nce.video.stream.mpegts.UdpStreamMonitor  289 | video-mpegts-stream  | initializing udp stream monitor: monitoredAddress=127.0.0.1, monitoredPort=50000, udpStreamProcessor=UdpStreamProcessor{defaultKlvHandler=org.codice.alliance.libs.klv.LoggingKlvHandler@2e8b423d, filenameGenerator=ListFilenameGenerator{filenameGeneratorList=[FileExtensionFilenameGenerator{extension='ts'}, DateTemplateFilenameGenerator{}, IllegalCharactersFilenameGenerator{}]}, filenameTemplate='mpegts-stream-%{date=yyyy-MM-dd_hh:mm:ss}', klvHandlerFactory=org.codice.alliance.libs.klv.KlvHandlerFactoryImpl@4fa0b69, klvProcessor=org.codice.alliance.libs.klv.ListKlvProcessor@1f424a5a, metacardTypeList=[ddf.catalog.data.impl.MetacardTypeImpl@50d8294c], packetBuffer=PacketBuffer{bytesWrittenToTempFile=0, incompleteFrameBytes=0}, rolloverCondition=BooleanOrRolloverCondition{firstCondition=ByteCountRolloverCondition{byteCountThreshold=10000000}, secondCondition=ElapsedTimeRolloverCondition{elapsedTimeThreshold=60000}}, stanag4609Processor=org.codice.alliance.libs.klv.Stanag4609ProcessorImpl@3e6604a0, metacardUpdateInitialDelay=2, parentMetacardUpdater=ListMetacardUpdater{metacardUpdaterList=[LocationMetacardUpdater{postUnionGeometryOperator=GeometryOperator{operatorList=[SimplifyGeometryFunction{distanceTolerance=Optional[0.1]}, NormalizeGeometry{}]}, preUnionGeometryOperator=org.codice.alliance.libs.klv.GeometryReducer@4a8823fb}, org.codice.alliance.video.stream.mpegts.metacard.TemporalStartMetacardUpdater@1e7dd880, org.codice.alliance.video.stream.mpegts.metacard.TemporalEndMetacardUpdater@1360418f, org.codice.alliance.video.stream.mpegts.metacard.ModifiedDateMetacardUpdater@1dad1447, FrameCenterMetacardUpdater{} LineStringMetacardUpdater{attributeName='frame-center-location', geometryOperator=GeometryOperator{operatorList=[SimplifyGeometryFunction{distanceTolerance=Optional[0.1]}, NormalizeGeometry{}, org.codice.alliance.libs.klv.GeometryReducer@16b17bf2]}}]}}

**Comment by glenhein** [comment]
Ben explained why the distance tolerance wasn't updated on the metatype page. A page refresh results in the correct value being displayed.

**Comment by glenhein** [comment]
the code build and runs as expected

**Comment by jlcsmith** [comment]
LGTM 👍

**Comment by jaymcnallie** [comment]
LGTM

**Pull request merged**